### ISSUE TITLE
Website is unusable on some lower spec hardware [nxos.org]

### ISSUE THREAD

**Issue by UriHerrera** [Nitrux/wordpress-sites-tracker#1]
Moved from: https://github.com/Nitrux/nitrux-website-html5/issues/7

CC @star-buck

> Please make the current website (I know this is an old repo, still putting this here) work without all that super-heavy Youtube Videos and Javascript madness.
It simply didnt ever finish initliazing loading here, and overall is a very discouraging experience and in stark contrast to the somewhat light-hearted and -weighed experience from nomad.
Thanks for taking care of this.

**Comment by UriHerrera** [comment]
I've updated the front page and removed the YT backgrounds and fade in effects on the columns. The titles and the separators have effects on their own. I've also optimized the images on the website and minified JS files that were slowing down the loading time of the site.

I've also added a Cloudflare plugin.

According to https://tools.pingdom.com/#!/dR1B1e/http://nxos.org the performance grade went from C/69 to A/93. Loading time went up from 24s to ~6s.
